server/api/controllers/forms/post-create-submission.js

```diff
@@ -153,6 +154,71 @@ module.exports = {
         });
       }
 
+      // Send alert email to leaders if setting is turned on
+      if (formRecord[0].alertType !== 'None') {
+        let emailRecipients;
+
+        // get email addresses of the recipients
+        if (formRecord[0].alertType === 'Custom') {
+          emailRecipients = formRecord[0].customAlertRecipients;
+        } else if (
+          ['LIFE Group', 'Lifestage', 'Campus'].includes(
+            formRecord[0].alertType
+          )
+        ) {
+          const recipientsDict =
+            await sails.helpers.forms.getFormAlertRecipients(formId);
+
+          // get the current season leadership teams
+          const today = new Date();
+
+          let fieldSelected;
+
+          switch (formRecord[0].alertType) {
+            case 'Campus':
+              fieldSelected = 'campus';
+              break;
+            case 'Lifestage':
+              fieldSelected = 'lifestage';
+              break;
+            case 'LIFE Group':
+              fieldSelected = 'lifeGroup';
+              break;
+          }
+
+          const recipientsEntries =
+            recipientsDict[submissionData[fieldSelected]];
+
+          for (const entry of recipientsEntries) {
+            if (
+              today >= new Date(entry.seasonFrom) &&
+              today <= new Date(entry.seasonTo)
+            ) {
+              emailRecipients = entry.leaderEmails;
+            }
+          }
+        }
+
+        // failsafe
+        if (emailRecipients.length <= 0) {
+          emailRecipients = ['web@hongkong.hmcc.net'];
+        }
+
+        // send notification email
+        await sails.helpers.sendTemplateEmail.with({
+          to: emailRecipients[0],
+          cc: emailRecipients,
+          subject: 'New Sign-up for ' + formRecord[0].formName,
+          template: 'form-notify-leader-success',
+          templateData: {
+            submissionTime: res.createdAt,
+            fullName: user.fullName || submissionData['fullName'],
```

Comment:
we should do error handling for these fields. if no full name, email, and phone number and user is not logged in, this section would crash